Week2/homework/js exercises/2-TheEvenOddReporter.js

```diff
@@ -0,0 +1,20 @@
+//Report whether or not a number is odd/even!
+
+//Create a for loop, that iterates from 0 to 20.
+
+//Create a conditional statement that checks 
+//if the value of the counter variable is odd or even.
+
+//If it's odd, log to the console 
+//The number [PUT_NUMBER_HERE] is odd!.
+
+//If it's even, log to the console 
+//The number [PUT_NUMBER_HERE] is even!.
+
+for (let i = 0; i < 21; i++){
+	if (i % 2 !== 0) {
+		console.log("The number " + i + " is ODD!");
+	}
+	else {console.log("The number " + i + " is even!");
+	}
+}
```

Comment:
Well done, works well using the modulo operator. Please note to use the right indentation to keep your code clean. So in this case. 
```
for (let i = 0; i < 21; i++){
	if (i % 2 !== 0) {
		console.log("The number " + i + " is ODD!");
	}
	else {
		console.log("The number " + i + " is even!");
	}
}
```